doc/source/ray-overview/installation.rst

```diff
@@ -370,10 +370,11 @@ To install Ray libraries, use ``pip`` as above or ``conda``/``mamba``.
 
 .. code-block:: bash
 
-  conda install -c conda-forge "ray-air"    # installs Ray + dependencies for Ray AI Runtime
+  conda install -c conda-forge "ray-data"   # installs Ray + dependencies for Ray Data
```

Comment:
Should we remove `ray-air` from conda? https://github.com/conda-forge/ray-packages-feedstock/blob/main/recipe/meta.yaml#L109